File: OdonSysBackEnd/Resource/Contract/Contract.Authentication/User/UpdateUserRequest.cs
using Utilities.Enums;

namespace Contract.Workspace.User
{
    public class UpdateUserRequest
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string Document { get; set; }
        public Country Country { get; set; }
    }
}

File: OdonSys-SPA/src/app/core/models/users/update-user-request.ts
import { Country } from '../../enums/country.enum';

export class UpdateUserRequest {
  constructor(
    public id: string,
    public name: string,
    public middleName: string,
    public lastName: string,
    public middleLastName: string,
    public document: string,
    public country: Country,
    public phone: string,
    public active: boolean
  ) {}
}
